### PR TITLE
Draft the display of the mote's data

### LINKED ISSUE #6: Draft the display of the mote's data

### PR DESCRIPTION
**Pull request by VVarnier**
close #6

### CODE REVIEW

**Comment by VVarnier** [comment]
MoteMeasureUI

File: IoTLab-Temperatures/IoTLab-Temperatures/MainPage.xaml.cpp [hunk 1]
```diff
@@ -72,11 +72,61 @@ void IoTLab_Temperatures::MainPage::LongitudeBox_TextChanged(
...
+
+
+void IoTLab_Temperatures::MainPage::RenderClosestMoteTemperature() {
+	TemperatureImage->Visibility = Windows::UI::Xaml::Visibility::Visible;
+	TemperatureTextBlock->Visibility = Windows::UI::Xaml::Visibility::Visible;
+	TemperatureRectangle->Visibility = Windows::UI::Xaml::Visibility::Visible;
+	TemperatureValueTextBlock->Visibility = Windows::UI::Xaml::Visibility::Visible;
+}
+
 double IoTLab_Temperatures::MainPage::ToDouble(Platform::String^ value) {
```
Comment (pBouillon) [inline IoTLab-Temperatures/IoTLab-Temperatures/MainPage.xaml.cpp]: This method should be deleted

File: IoTLab-Temperatures/IoTLab-Temperatures/MainPage.xaml.cpp [hunk 2]
```diff
@@ -72,11 +72,61 @@ void IoTLab_Temperatures::MainPage::LongitudeBox_TextChanged(
...
+
+void IoTLab_Temperatures::MainPage::RenderClosestMoteHumidity() {
+	HumidityImage->Visibility = Windows::UI::Xaml::Visibility::Visible;
+	HumidityTextBlock->Visibility = Windows::UI::Xaml::Visibility::Visible;
+	HumidityRectangle->Visibility = Windows::UI::Xaml::Visibility::Visible;
+	HumidityValueTextBlock->Visibility = Windows::UI::Xaml::Visibility::Visible;
+}
+
+void IoTLab_Temperatures::MainPage::RenderClosestMoteMeasure() {
+
```
Comment (pBouillon) [inline IoTLab-Temperatures/IoTLab-Temperatures/MainPage.xaml.cpp:127]: blank line

File: IoTLab-Temperatures/IoTLab-Temperatures/MainPage.xaml.cpp [hunk 3]
```diff
@@ -89,6 +139,15 @@ void IoTLab_Temperatures::MainPage::UpdateValidateButtonValidity()
 }
 
 
+void IoTLab_Temperatures::MainPage::UpdateDisplayedMeasures(double battery, double brightness, double humidity, double temperature) {
+
```
Comment (pBouillon) [inline IoTLab-Temperatures/IoTLab-Temperatures/MainPage.xaml.cpp]: blank line

**Review by pBouillon — CHANGES_REQUESTED** [review]

**Review by pBouillon — APPROVED** [review]

**Pull request merged**